testutil/testproxy/relayerproxy.go

```diff
@@ -338,14 +339,22 @@ func GetRelayResponseError(t *testing.T, res *http.Response) (errCode int32, err
 
 	relayResponse := &servicetypes.RelayResponse{}
 	err = relayResponse.Unmarshal(responseBody)
+	require.NoError(t, err)
+
+	// If the relayResponse basic validation fails then consider the payload as an error.
+	if err := relayResponse.ValidateBasic(); err != nil {
+		return -32000, string(relayResponse.Payload)
```

Comment:
What is the magic -32000 number? Either #PUC or move it to a constant w/ a comment.